.changelog/5962.cfg.md

```diff
@@ -23,4 +23,6 @@ The following configuration options have been added:
    used to fetch runtime bundles,
 
 - `runtime.repositories` is the list of global URLs used to fetch
-   runtime bundles.
+   runtime bundles,
+
+- `runtime.max_bundle_size_bytes` is the maximum allowed bundle size in bytes.
```

Comment:
We also have the `ParseSizeInBytes` helper to support standard suffixes and then this can be called `max_bundle_size` (without the `_bytes`). See existing config options.